tests/integration/test_show.py

```diff
@@ -50,25 +50,23 @@ def test_columns(c):
     assert_eq(result_df, expected_df)
 
 
-@pytest.mark.skip(reason="WIP DataFusion")
 def test_wrong_input(c):
     with pytest.raises(KeyError):
         c.sql('SHOW COLUMNS FROM "wrong"."table"')
-    with pytest.raises(AttributeError):
+    with pytest.raises(ParsingException):
         c.sql('SHOW COLUMNS FROM "wrong"."table"."column"')
     with pytest.raises(KeyError):
         c.sql(f'SHOW COLUMNS FROM "{c.schema_name}"."table"')
     with pytest.raises(AttributeError):
         c.sql('SHOW TABLES FROM "wrong"')
 
 
-@pytest.mark.skip(reason="WIP DataFusion")
 def test_show_tables_no_schema(c):
     c = Context()
 
     df = pd.DataFrame({"id": [0, 1]})
     c.create_table("test", df)
 
-    actual_df = c.sql("show tables").compute()
+    actual_df = c.sql("show tables from root").compute()
```

Comment:
Should this be a requirement? If `show tables` is encountered should the parser just insert the default value of `root` into the `ShowColumnsPlanNode` instead? Or its lack of presence cause the native parser path to be taken? If that is the case I think this is fine to leave.